.commitlintrc.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
   ignores: [
     (commit) => commit.includes('[ci-skip]'),
     (commit) => commit.includes('Pull request'),
+    (commit) => commit === 'Updated styles and tokens',
```

Comment:
I will file an issue for this.

Reply:
File it, but we won't be able to fix till Supernova allows us to change it.

Reply:
Sure, it's in the backlog.